x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/types.ts

```diff
@@ -37,6 +37,7 @@ export interface SignalAlertParams {
   references: string[];
   savedId: string | undefined;
   severity: string;
+  size?: number;
```

Comment:
The size property represents the size of search result returned from each search_after. This would (maybe?) allow the user to modify how many and simultaneously how large of a search_after query they want to run on their Elasticsearch cluster when re-indexing. From the sounds of it, I think `maxSignals` has a slightly different use case than from `size`?

Reply:
I will update the size property to be defined with the ` | undefined` though.